src/components/inputs/money-input/money-input.js

```diff
@@ -31,6 +31,11 @@ const createCurrencySelectStyles = ({
     }),
     singleValue: base => ({
       ...base,
+      marginLeft: '0',
+      marginRight: '0',
+      maxWidth: '100%',
+      left: '50%',
+      transform: 'translate(-50%, -50%)',
```

Comment:
this fixes the ellipsis problem.

Reply:
Let's add a code comment so the next person reading this knows what's up.

Reply:
Just figured this was easiest way to fix the problem without changing the look or width of the input. Just replaced the margin on the left and the right, made the max width 100% instead, and centre aligned it.

Reply:
I found a more natural way to do this without `transform`. I'll push the commit onto this branch. Setting `maxWidth: 'initial'` fixes the problem on its own luckily.